monitoring/dashboards/mongodb.json

```diff
@@ -0,0 +1,1909 @@
+{
+  "__inputs": [
```

Comment:
This section is missing the `namespace` contant input. we want to enforce the filter on only mongodb in zenko namespace.